Library/Homebrew/blacklist.rb

```diff
@@ -1,16 +1,5 @@
 def blacklisted? name
   case name.downcase
-  when 'screen', /^rubygems?$/ then <<-EOS.undent
```

Comment:
No, I think we should put it where `search_tap`'s result gets print. The reason is this method is calling somewhere else e.g. audit.rb.